src/main/java/com/sfuentes/Stationery.java

```diff
@@ -0,0 +1,50 @@
+package com.sfuentes;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class Stationery {
+
+  Map<String, Integer> requested;
+
+  public Stationery() {
+    this.requested = new HashMap<>();
+  }
+
+  public void addRequest(String typeOfPrinting, int numOfCopies) {
+    if (requested.containsKey(typeOfPrinting))
+      requested.put(typeOfPrinting, requested.get(typeOfPrinting) + numOfCopies);
+    else
+      requested.put(typeOfPrinting, numOfCopies);
+  }
+
+
+  public List<String> verifyRequest() {
+
+    Map<String, Integer> lowerCaseMap = requested.entrySet()
+        .parallelStream()
+        .collect(Collectors
+            .toMap(entry -> entry.getKey()
+                .toLowerCase(), Map.Entry::getValue));
```

Comment:
I think you don't need to go through the map to lowercase the keys. If you really need to do that you can just lowercase the keys in the addRequest method before you put them or just do it in the if you have below when comparing.

Reply:
fixed.